sdk/storage/azure-storage-file-share/HISTORY.md

```diff
@@ -1,6 +1,16 @@
 # Release History
 
-## 12.0.1 (Unreleased)
+## 12.1.0
+
+**New features**
+- Added support for the 2019-07-07 service version.
+- Added support for file leases. Including adding both sync and async `ShareFileLeaseClient`, accepting leaseIds for operations that support leases.
```

Comment:
`ShareFileLeaseClient` was introduced to both sync and async versions of the SDK, which allows users to perform operations on file leases.